ui/lib/core/addon/templates/components/replication-action-promote.hbs

```diff
@@ -54,7 +54,13 @@
             Primary cluster address <em class="is-optional">(optional)</em>
           </label>
           <div class="control">
-            {{input class="input" id="primary_cluster_addr" name="primary_cluster_addr" value=primary_cluster_addr}}
+            {{input
+              class="input"
+              id="primary_cluster_addr"
+              name="primary_cluster_addr"
+              value=primary_cluster_addr
+              enter=(action "onSubmit" "promote" model.replicationAttrs.modeForUrl (hash dr_operation_token=dr_operation_token primary_cluster_addr=primary_cluster_addr force=force))
```

Comment:
allowing the enter button to submit when you are in the input of the primary_cluster_addr